### PR TITLE
add alert policies: check disk available space

### PR DESCRIPTION
**Pull request by mfuuzy**

### CODE REVIEW

**Review by rainchen — CHANGES_REQUESTED** [review]
please rewrite your commit message in English

File: server/config/alert.rules [hunk 1]
```diff
@@ -79,3 +79,13 @@ groups:
       summary: "Directory growth per second overflow"
       description: "Directory for 1 minutes avg growth of per second exceeds 5 bytes | [exporter info http://{{ $labels.instance }}/metrics]"
       rate: "{{ $value }} bytes/s"
+
+  # DISK 不足告警
+  - alert: DISK_Warning
+    expr: (Node_Get_DiskInfo_UsedSize / ( Node_Get_DiskInfo_UsedSize + Node_Get_DiskInfo_FreeSize)) * 100   > 80
```
Comment (rainchen) [inline server/config/alert.rules]: wrong expr, please check https://github.com/citahub/cita-monitor/issues/151
should use `FreeSize percentage`  < 10

Comment (mfuuzy) [reply server/config/alert.rules]: 可以了

File: server/config/alert.rules [hunk 2]
```diff
@@ -79,3 +79,13 @@ groups:
...
       description: "Directory for 1 minutes avg growth of per second exceeds 5 bytes | [exporter info http://{{ $labels.instance }}/metrics]"
       rate: "{{ $value }} bytes/s"
+
+  # DISK 不足告警
+  - alert: DISK_Warning
+    expr: (Node_Get_DiskInfo_FreeSize / ( Node_Get_DiskInfo_UsedSize + Node_Get_DiskInfo_FreeSize)) * 100 < 10
+    labels:
+      severity: high
+    annotations:
+      summary: "DISK useage overflow"
```
Comment (rainchen) [inline server/config/alert.rules]: 磁盘空间没有 overflow 的说法

**Review by rainchen — CHANGES_REQUESTED** [review]
please update

File: server/config/alert.rules [hunk 3]
```diff
@@ -79,3 +79,13 @@ groups:
       summary: "Directory growth per second overflow"
       description: "Directory for 1 minutes avg growth of per second exceeds 5 bytes | [exporter info http://{{ $labels.instance }}/metrics]"
       rate: "{{ $value }} bytes/s"
+
+  # DISK 不足告警
+  - alert: DISK_Warning
```
Comment (rainchen) [inline server/config/alert.rules]: 把 `DISK` 改为 `Disk`

File: server/config/alert.rules (hunk 2 above)
Comment (rainchen) [reply server/config/alert.rules]: summary 内容改 `Disk is running out of free space`

**Review by rainchen — APPROVED** [review]

**Comment by rainchen** [comment]
@mfuuzy must change base branch to develop

**Review by rainchen — CHANGES_REQUESTED** [review]
wrong base branch

**Review by rainchen — APPROVED** [review]

**Pull request merged**